bbn-test-4/finality-providers/README.md

```diff
@@ -96,7 +101,7 @@ previous generated.
 
 ```shell
 stakercli transaction create-phase1-staking-transaction \
-  --staker-pk <your_generated_pub_key> \
+  --staker-pk <your_generated_staker_pub_key> \
```

Comment:
Let's also add a note that (1) it can be different for the finality provider pub key or the same, depending on where the funds have been sent and (2) it is recommended to keep the finality provider keyring offline, so use a separate account to create a transaction?

Reply:
https://github.com/babylonchain/networks/pull/193/commits/416ffd8668f420f8c19aeba8d44144e06ccd2b49